src/jira/client/axios.ts

```diff
@@ -114,7 +114,7 @@ function getSuccessMiddleware(logger) {
 				},
 				`Jira request: ${response.config.method.toUpperCase()} ${
 					response.config.originalUrl
-				} - ${response.status} ${response.statusText}`
+				} - ${response.status} ${response.statusText}\\n${JSON.stringify(response.data)}\`);`
```

Comment:
the JSON.stringify bit doesn't look right.  First, you have a `\\n` which will escape and show literally `\n` in the string.  Then you have an errant escaped tilde and a `);` in the string.

Reply:
Copying/pasting fail 🤦‍♀️